### ISSUE TITLE
Builds getting killed by RTD

### ISSUE THREAD

**Issue by zxiiro** [rtfd/readthedocs.org#2939]
## Details

* Project URL: https://readthedocs.org/projects/opendaylight
* Build URL (if applicable): https://readthedocs.org/projects/opendaylight/builds/5539621/
* Read the Docs username (if applicable): zxiiro

## Expected Result

Build successful

## Actual Result

Our builds keep getting killed during the build process. It seems something is killing our process perhaps? Hoping to understand why this is happening. Thanks!

**Comment by nilok** [comment]
More information from our internal mailing list here:
https://lists.opendaylight.org/pipermail/documentation/2017-June/001206.html

> This one took 453 seconds and generated html:
> https://readthedocs.org/projects/opendaylight/builds/5535226/ (git hash 2faed1703e572b2b2d9632256273b16fe7d3a728)
>
> The next build on stable-carbon (and all following ones) take less than 60 seconds and don't generate anything:
> https://readthedocs.org/projects/opendaylight/builds/5535694/ (git hash c05fdb7191434e84ccfccbc218f2cc07397c1c30)
>
> This shows the diff between the repos at those times:
> https://github.com/opendaylight/docs/compare/2faed1703e572b2b2d9632256273b16fe7d3a728...c05fdb7191434e84ccfccbc218f2cc07397c1c30
>
> Looking through those changes the only thing I can imagine breaking anything would be the :doc: links to the files in submodules, but there are some of those in commit 2faed17 too…

**Comment by zxiiro** [comment]
We believe we tracked down the problem to our process forking script and fixed it with this patch https://git.opendaylight.org/gerrit/58840

Theory is we maybe be forking too many subprocesses and RTD is killing the build. Closing this issue.

I am interested in knowing of the RTD servers have GNU parallel installed though. We could improve that script using parallel so that it more smartly creates subprocesses.
